lib/cli.js

```diff
@@ -59,7 +59,7 @@ var yargs = require("yargs"),
 
     HOST_MESSAGE = "If no hosts are given, a screenshot from localhost is displayed.",
 
-    COMMANDS = ["search", "path", "ia", "group"],
+    COMMANDS = ["search", "path", "ia", "group", "build-group"],
```

Comment:
Do you need a new command for this? What if you allow it to happen under the group command? First look for a script, if no script is found, look for a group file?

I think it might be an easier api to use if we didn't need to remember how to specify group/build-group.